make-webpack-config.js

```diff
@@ -25,6 +26,7 @@ module.exports = function(options){
   }
 
   return {
+    devtool: isProd ? 'hidden-source-map' : 'eval',
```

Comment:
Well considering that redbox is most likely indented as a devtool, then I would not bother emitting any source map for a production really. It just slows things down.